client/my-sites/plugins/marketplace/marketplace-domain-upsell/index.tsx

```diff
@@ -0,0 +1,209 @@
+/**
+ * External dependencies
+ */
+import React, { useEffect, useState } from 'react';
+import { useSelector } from 'react-redux';
+import page from 'page';
+import DomainPicker from '@automattic/domain-picker';
+import { ResponseCart, useShoppingCart } from '@automattic/shopping-cart';
+import { DomainSuggestions } from '@automattic/data-stores';
+import { ThemeProvider } from 'emotion-theming';
+import styled from '@emotion/styled';
+import { translate } from 'i18n-calypso';
+import classnames from 'classnames';
+import { isDesktop } from '@automattic/viewport';
+
+/**
+ * Internal dependencies
+ */
+import CalypsoShoppingCartProvider from 'calypso/my-sites/checkout/calypso-shopping-cart-provider';
+import { domainRegistration } from 'calypso/lib/cart-values/cart-items';
+import {
+	MARKETPLACE_FLOW_ID,
+	ANALYTICS_UI_LOCATON_MARKETPLACE_DOMAIN_SELECTION,
+} from 'calypso/my-sites/plugins/marketplace/constants';
+import { getSelectedSite } from 'calypso/state/ui/selectors';
+import { getProductsList, isProductsListFetching } from 'calypso/state/products-list/selectors';
+import { fillInSingleCartItemAttributes } from 'calypso/lib/cart-values';
+import Masterbar from 'calypso/layout/masterbar/masterbar';
+import Item from 'calypso/layout/masterbar/item';
+import getPreviousPath from 'calypso/state/selectors/get-previous-path';
+import { HorizontalRule } from '@wordpress/components';
+import ExternalLink from 'calypso/components/external-link';
+
+/**
+ * Style dependencies
+ */
+import 'calypso/my-sites/plugins/marketplace/marketplace-domain-upsell/style.scss';
+import MarketplaceShoppingCart from 'calypso/my-sites/plugins/marketplace/components/marketplace-shopping-cart';
+import theme from 'calypso/my-sites/plugins/marketplace';
+
+const MarketplaceHeaderTitle = styled.h1`
+	font-size: 2rem;
+	margin-bottom: 8px;
+`;
+const MarketplaceHeaderSubTitle = styled.h2`
+	font-size: 0.875rem;
+	margin-bottom: 15px;
+`;
+
+function MarketplaceDomainUpsellHeader() {
+	return (
+		<div>
+			<MarketplaceHeaderTitle className="marketplace-domain-upsell__title wp-brand-font">
+				{ translate( 'Choose a domain' ) }
+			</MarketplaceHeaderTitle>
+			<MarketplaceHeaderSubTitle>
+				{ translate(
+					'Yoast SEO requires a top level domain to index your site and help you rank higher.'
+				) }
+			</MarketplaceHeaderSubTitle>
+			<ExternalLink
+				href="https://yoast.com/domain-names-seo"
+				target="_blank"
+				rel="noopener noreferrer"
+				icon={ true }
```

Comment:
`={ true }` is redundant for props.
```suggestion
				icon
```